### PR TITLE
Implement bash autocompletion, improve serve command

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️/❌
| Fixed issues  | #138

### CODE REVIEW

File: src/Command/Serve.php [hunk 1]
```diff
@@ -34,17 +35,32 @@ final class Serve extends Command
...
     }
 
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        if ($input->mustSuggestArgumentValuesFor('address')) {
+            $suggestions->suggestValues(['localhost', '127.0.0.1', '0.0.0.0']);
+            return;
+        }
+
+        $suggestions->suggestOptions($this->getDefinition()->getOptions());
```
Comment (vjik) [inline src/Command/Serve.php:61]: Seems, autocompletion good work without this line.

File: src/Command/Serve.php [hunk 2]
```diff
@@ -103,7 +119,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return ExitCode::OK;
         }
 
-        passthru('"' . PHP_BINARY . '"' . " -S $address -t \"$documentRoot\" $router");
+        passthru('"' . PHP_BINARY . '"' . " -S $address -t $documentRoot $router");
```
Comment (vjik) [inline src/Command/Serve.php:122]: Can there be problems if `$documentRoot` contains spaces?

File: src/Command/Serve.php [hunk 3]
```diff
@@ -69,9 +85,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         /** @var string $env */
         $env = $input->getOption('env');
 
-        $documentRoot = getcwd() . '/' . $docroot; // TODO: can we do it better?
+        $documentRoot = $this->aliases->get('@root/' . $docroot);
```
Comment (vjik) [inline src/Command/Serve.php:88]: Alias can not contain `@root` . Can we do without aliases and don't add new dependency?

File: src/Command/Serve.php (hunk 2 above)
Comment (rustamwin) [reply src/Command/Serve.php:122]: Good catch.

File: src/Command/Serve.php (hunk 1 above)
Comment (rustamwin) [reply src/Command/Serve.php:61]: Yes, checked.

**Review by samdark — APPROVED** [review]

**Pull request merged**